src/crypto/OutgoingRoomKeyRequestManager.js

```diff
@@ -302,7 +302,6 @@ export class OutgoingRoomKeyRequestManager {
                                 "Error sending room key request cancellation;"
                                 + " will retry later.", e,
                             );
-                            this._startTimer();
```

Comment:
Retry isn't needed anymore?

Reply:
Mmm, good point - they would be sent the next time the queue was sent which would be on the next sync, but I guess keeping it closer to the original behaviour is probably better for now.